CONTRIBUTING.md

```diff
@@ -128,42 +120,47 @@ submit a pull request:
    to other methods available in pydicom.
 
 -  Documentation and high-coverage tests are necessary for enhancements to be
-   accepted. Bug-fixes shall be provided with 
+   accepted. Bug-fixes shall be provided with
    [regression tests](https://en.wikipedia.org/wiki/regression_testing) that
    fail before the fix. For new features, the correct behavior shall be
-   verified by feature tests. A good practice to write sufficient tests is 
+   verified by feature tests. A good practice to write sufficient tests is
    [test-driven development](https://en.wikipedia.org/wiki/Test-driven_development).
 
 You can also check for common programming errors and style issues with the
 following tools:
 
--  Code with good unittest **coverage** (current coverage or better), check
- with:
+-  The test coverage should be at least as good as the current coverage and any
+new lines of code must be covered:
 
   ```bash
   $ pip install pytest pytest-cov
-  $ py.test --cov=pydicom path/to/test_for_package
+  $ cd pydicom/
+  $ py.test --cov=pydicom tests/
```

Comment:
better use `pytest`, `py.test` is long deprecated

Reply:
Done